src/main/scala/autoversion/Keys.scala

```diff
@@ -1,17 +1,14 @@
 package autoversion
 
-import autoversion.model.{Commit, Tag}
+import autoversion.model.{Commit, ConventionalCommit, Tag}
 import sbt.{settingKey, taskKey}
 import sbtrelease.Version.Bump
 
-import scala.util.matching.Regex
-
 object Keys {
-  val latestTag         = taskKey[Option[Tag]]("Latest semver version from Git tags.")
-  val tagNameCleaner    = settingKey[String => String]("Cleans the git tag to extract only the version.")
-  val unreleasedCommits = taskKey[Seq[Commit]]("Commits since the latest tagged release.")
-  val suggestedBump     = taskKey[Bump]("Version bump computed by sbt-autoversion")
-  val bugfixRegexes     = settingKey[Seq[Regex]]("Regex that commit messages must follow to use a bugfix bump.")
-  val minorRegexes      = settingKey[Seq[Regex]]("Regex that commit messages must follow to use a minor bump.")
-  val majorRegexes      = settingKey[Seq[Regex]]("Regex that commit messages must follow to use a major bump.")
+  val autoVersionLatestTag         = taskKey[Option[Tag]]("Latest semver version from Git tags.")
```

Comment:
The key prefixes are implied by package scope, but because they are exposed by `autoImport`, if they're not themselves prefixed they may conflict with other keys from other plugins. It's called out in [plugin best practices](https://www.scala-sbt.org/1.x/docs/Plugins-Best-Practices.html#Key+naming+convention%3A+Use+prefix).

It is backwards breaking though, yeah. My thinking is that, since this PR changes the logic pretty significantly, a major version bump is probably in order either way.

Reply:
Indeed the keyspace collision rationale did occur to be some time after making this comment - I didn't realize it was called out in the plugin best practices as well, good call out.  Agreed this is as good a time as any to make that change.